toolbox

```diff
@@ -751,6 +752,19 @@ create()
         spinner_directory=""
     fi
 
+    if [ "$(readlink /home)" = var/home ] 2>&3; then
+	need_home_link=true;
+	echo "$base_toolbox_command: /home is a symlink to /var/home" >&3
+    else
+	need_home_link="";
+    fi
+
+    REALHOME=$HOME
+
+    if [ $need_home_link ] && [ ! "${HOME##/home*}" ]; then
+	REALHOME=/var$REALHOME
```

Comment:
It would be better to use `readlink --canonicalize "$HOME"` instead. If nothing else, it's a lot more readable. :)